docs/static/releasenotes.asciidoc

```diff
@@ -4,11 +4,31 @@
 coming[8.0.0]
 
 This section summarizes the changes in the following releases:
```

Comment:
```suggestion
This section summarizes the changes in the following releases:

```

Reply:
This return is required. Otherwise, the list renders as a paragraph. (See preview: https://logstash_13598.docs-preview.app.elstc.co/guide/en/logstash/8.0/releasenotes.html).